lib/delta.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+var util = require('util');
```

Comment:
modern Javascript style strongly encourages using const/let over the more archaic var.